docs/digger/admin/digger-nagios-monitoring.asciidoc

```diff
@@ -0,0 +1,7 @@
+== Digger Nagios Monitoring
+Nagios is provisioned by default by the Digger Ansible playbook. The Nagios instance will perform a number of checks (WHEN???):
```

Comment:
If when means when will Nagios perform the checks, this will happen periodically. I'm not sure of the default timings or whether our nagios checks will have configurable check frequencies.

From these docs it looks like every 60 minutes by default? https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/objectdefinitions.html

@laurafitzgerald do you know if we allow configuration like this in nagios?